Description: Build against udisks1 instead of 2
Author: Leszek Lesner <leszek@zevenos.com>

--- kde4libs-4.12.0.orig/CMakeLists.txt
+++ kde4libs-4.12.0/CMakeLists.txt
@@ -39,7 +39,7 @@ if(KDE_PLATFORM_FEATURE_DISABLE_DEPRECAT
 endif(KDE_PLATFORM_FEATURE_DISABLE_DEPRECATED)
 
 ############### Give the user the option to build the udisks2 solid backend instead of the udisks backend ###############
-option(WITH_SOLID_UDISKS2 "Enable the udisks2 solid backend instead" "ON")
+option(WITH_SOLID_UDISKS2 "Enable the udisks2 solid backend instead" "OFF")
 
 ############### Give the user the option to build the deprecated WMI solid backend instead of the new win backend ###############
 option(WITH_SOLID_WMI "Enables the deprecated WMI backend on Windows")
